package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "11.2.3",
+  "version": "11.2.4",
```

Comment:
I think we can consider this new functionality.. 11.3.0?

Reply:
I am ok after that is addressed..

Reply:
Done